platform-api/src/main/java/io/datacater/core/stream/StreamEndpoint.java

```diff
@@ -43,8 +43,10 @@ public Uni<StreamEntity> getStream(@PathParam("uuid") UUID uuid) {
   @GET
   @Path("{uuid}/inspect")
   public Uni<List<StreamMessage>> inspectStream(
-      @PathParam("uuid") UUID uuid, @DefaultValue("100") @QueryParam("limit") Long limit) {
-    return streamsUtil.getStreamMessages(uuid, limit);
+      @PathParam("uuid") UUID uuid,
+      @DefaultValue("100") @QueryParam("limit") Long limit,
+      @DefaultValue("SEQUENCED") @QueryParam("sampleMethod") SampleMethod method) {
```

Comment:
I would also rename `method` to `sampleMethod` here, to align with the naming of the query parameter.